src/app/components/theme-editor/theme-editor.html

```diff
@@ -17,16 +17,15 @@
         </div>
         <div ng-show="editor.busy"><i class="fa fa-spinner fa-pulse fa-2x"></i></div>
       </div>
-
       <uib-tabset class="top-buffer-2">
         <uib-tab heading="Theme">
           <div class="row top-buffer-1">
             <div class="form-group col-md-6" ng-repeat="(color, value) in theme">
               <label>{{labels[color]}} <br/><small class="text-muted">{{color}}</small></label>
-              <input type="color" class="form-control"
-                     ng-model="theme[color]"
-                     ng-model-options="{ debounce: 300 }"
-                     ng-change="editor.update()">
+              <spectrum-colorpicker
+                ng-model="theme[color]" ng-model-options="{ debounce: 300 }" ng-change="editor.update()" palette
```

Comment:
Are you sure you need the attribute `palette` here? Is it used to display it?

Reading the doc, it seems to me it is used to define the colors in the palette.

Reply:
It is unnecessary. I just pushed a cleanup